pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -701,6 +722,12 @@ func (oe *operationExecutor) generateMountVolumeFunc(
 				volumeToMount.VolumeSpec.Name(),
 				volumeToMount.PodName,
 				volumeToMount.Pod.UID)
+			oe.recorder.Eventf(volumeToMount.Pod, api.EventTypeNormal, kevents.SuccessfulMountVolume,
```

Comment:
True on the unmount. Even if it is once per volume though, every single pod generating at a minimum two new events (possibly many more) seems like something to be careful with. I remember the days when events were overloading etcd